### PR TITLE
fix: fix and update docs and readmes for reciprocalrank

### PR DESCRIPTION
**Pull request by JoanFM**

### CODE REVIEW

File: evaluators/rank/ReciprocalRankEvaluator/manifest.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ description: |
 author: Jina AI Dev-Team (dev-team@jina.ai)
 url: https://jina.ai
 vendor: Jina AI Limited
-documentation: https://github.com/jina-ai/jina-hub
+documentation: https://github.com/jina-ai/jina-hub/blob/master/evaluators/rank/NdcgEvaluator/README.md
 version: 0.0.5
```
Comment (bwanglzu) [inline evaluators/rank/ReciprocalRankEvaluator/manifest.yml:10]: version needs to be updated?

**Review by bwanglzu — CHANGES_REQUESTED** [review]

File: evaluators/rank/ReciprocalRankEvaluator/manifest.yml (hunk 1 above)
Comment (JoanFM) [reply evaluators/rank/ReciprocalRankEvaluator/manifest.yml:10]: good point, can u add the suggestion?

Comment (bwanglzu) [reply evaluators/rank/ReciprocalRankEvaluator/manifest.yml:10]: @JoanFM I can't because it's not an edited line.

Comment (nan-wang) [reply evaluators/rank/ReciprocalRankEvaluator/manifest.yml]: done

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Review by theUnkownName — APPROVED** [review]

**Pull request merged**